src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -92,29 +96,43 @@ winrt::com_ptr<Profile> Profile::CopySettings(winrt::com_ptr<Profile> source)
     profile->_Padding = source->_Padding;
     profile->_Commandline = source->_Commandline;
     profile->_StartingDirectory = source->_StartingDirectory;
-    profile->_BackgroundImagePath = source->_BackgroundImagePath;
-    profile->_BackgroundImageOpacity = source->_BackgroundImageOpacity;
-    profile->_BackgroundImageStretchMode = source->_BackgroundImageStretchMode;
+    copyDefaultAppearance->_BackgroundImagePath = sourceDefaultAppearance->_BackgroundImagePath;

```

Comment:
Same as the others, can we use `AppearanceConfig::CopyAppearance` to de-dupe some code here?